x/feegrant/keeper/keeper.go

```diff
@@ -100,14 +98,14 @@ func (k Keeper) IterateAllGranteeFeeAllowances(ctx sdk.Context, grantee sdk.AccA
 	stop := false
 	for ; iter.Valid() && !stop; iter.Next() {
 		bz := iter.Value()
-		var grant types.FeeAllowanceGrant
 
-		err := k.cdc.UnmarshalBinaryBare(bz, &grant)
+		var feeGrant types.FeeAllowanceGrant
+		err := feeGrant.Unmarshal(bz)
```

Comment:
Why are we still calling `Unmarshal` directly?

Reply:
Oh, this should be replaced. Updating now